apps/staking/.env.development

```diff
@@ -1,3 +1,4 @@
 NEXT_PUBLIC_ENVIRONMENT=development
 NEXT_PUBLIC_UNLEASH_PROXY_HOST=https://preview.unleash.cartesi.io/proxy
-NEXT_PUBLIC_UNLEASH_PROXY_CLIENT_KEY=bc1b71da9932b90ccd788d75203fb598c3640cabed3ccccdffd024f341a79d72
\ No newline at end of file
+NEXT_PUBLIC_UNLEASH_PROXY_CLIENT_KEY=bc1b71da9932b90ccd788d75203fb598c3640cabed3ccccdffd024f341a79d72
+NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID=ba5de27656b7af8e2dab344f790ad81a
```

Comment:
Hi @dandheedge, 
Could you remove that? That is your WalletConnect project-id. I will create one for Staking and include the config on Vercel. To avoid versioning your configs, create and use a `.env.development.local` file, as the Staking README explains.